test/java/config/fortify-client.properties

```diff
@@ -0,0 +1,3 @@
+fortify.client.releaseId=65418
+fortify.exclude.pattern=*.jar|*.zip|*.tar|/build|/.gradle|/.git|/node_modules|/test|/dist
```

Comment:
Please remove this line, as otherwise this overwrites the default, more detailed exclusion list.

Reply:
Removed